PSSwagger/PSSwagger.Resources.psd1

```diff
@@ -89,5 +89,6 @@ ConvertFrom-StringData @'
     InvalidHeaderFileExtension=Header '{0}' file extension should be '.txt'.
     InvalidHeaderFilePath=The specified value '{0}' for Header parameter is should be a valid file path.
     HeaderContentTwoHyphenWarning=The specified Header content has '--', replacing '--' with '=='.
+    PSScriptAnalyzerMissing=PSScriptAnalyzer specified as the formatter, but the module is not available on this machine. Install PSScriptAnalyzer from PowerShellGallery to use this as a formatter.
```

Comment:
Consider adding 'Install-Module -Name PSScriptAnalyzer -Repository PSGallery' in the message so that user will copy the command and run it.